package.json

```diff
@@ -11,17 +11,61 @@
     "url": "https://github.com/sponsors/tannerlinsley"
   },
   "main": "lib/index.js",
+  "module": "lib/index.mjs",
+  "types": "lib/index.d.ts",
   "unpkg": "dist/react-query.development.js",
-  "types": "types/index.d.ts",
-  "module": "es/index.js",
   "sideEffects": [
-    "es/index.js",
-    "es/reactjs/index.js",
-    "es/reactjs/setBatchUpdatesFn.js",
     "lib/index.js",
+    "lib/index.mjs",
     "lib/reactjs/index.js",
-    "lib/reactjs/setBatchUpdatesFn.js"
+    "lib/reactjs/index.mjs",
+    "lib/reactjs/setBatchUpdatesFn.js",
+    "lib/reactjs/setBatchUpdatesFn.mjs",
+    "src/reactjs/setBatchUpdatesFn.ts"
   ],
+  "exports": {
+    ".": {
+      "import": "./lib/index.mjs",
+      "default": "./lib/index.mjs"
+    },
+    "./package.json": "./package.json",
+    "./core": {
+      "import": "./lib/core/index.mjs",
+      "default": "./lib/core/index.js"
+    },
+    "./reactjs": {
+      "import": "./lib/reactjs/index.mjs",
+      "default": "./lib/reactjs/index.mjs"
+    },
+    "./devtools": {
+      "development": {
```

Comment:
will we still be able to import from react-query/devtools/development to do e.g. this:

https://twitter.com/tannerlinsley/status/1392857647037091849

?

Reply:
No, although I could add support for that.

Reply:
added support for it